static/sidebar/css/modal/styles.css

```diff
@@ -128,12 +128,33 @@ body  {
   font-size: 14px;
   text-decoration: none;
   margin: 10px 0;
+  color: #1F1D26;
+}
+
+.tvp-product-title .tvp-tooltip {
```

Comment:
tooltip will be a custom thing for Bleeping, please just remove it from this branch so it doesn't stop your fixes to go live